source/features/show-open-prs-of-forks.tsx

```diff
@@ -60,7 +60,7 @@ async function getPRs(): Promise<[prCount: number, url: string] | []> {
 		return [count, `/${forkedRepo}/pull/${firstPr!}`];
 	}
 
-	return [count, `/${forkedRepo}/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+author%3A${getUsername()!}`];
+	return [count, `/${forkedRepo}/pulls?q=${encodeURIComponent('is:pr is:open sort:updated-desc author:@me')}`];
```

Comment:
Might as well change it to use URLSearchParams or SearchQuery 👍

Reply:
There's only one parameter so `URLSearchParams` doesn't make much difference and `SearchQuery` is `URLSearchParams` under the hood.

Reply:
If a "lint" change is made to this line, then it should be linted fully
```suggestion
	const url = new URL(`/${forkedRepo}/pulls`);
	url.searchParams.set('q', 'is:pr is:open sort:updated-desc author:@me');
	return [count, url.href];
```

Building search parameters with strings is generally not advised on the repo